protected/humhub/modules/notification/components/BaseNotification.php

```diff
@@ -129,6 +129,16 @@ protected function category()
         return null;
     }
 
+    /**
+     * Checks if notification is still actual before sending
+     *
+     * @return bool
+     */
+    public function isActual()
```

Comment:
I also think we should improve the wording here.

What do you think about `$notification->isValid()`. This method should check if the notification should still exists, if not, the notification should be deleted.

Reply:
Please also add the as PHPDoc `@since 1.15`